src/pykoala/instruments/koala_ifu.py

```diff
@@ -208,14 +215,20 @@ def koala_rss(path_to_file):
     info['fib_dec'] = np.rad2deg(koala_header['DECCEN']) + koala_fibre_table.data['Delta_DEC'] / 3600
     info['airmass'] = airmass_from_header(koala_header)
     # Read RSS file into a PyKoala RSS object
-    rss = read_rss(path_to_file, wcs=koala_wcs,
-                   bad_fibres_list=bad_fibres_list,
+    rss = read_rss(path_to_file, 
+                   wcs=koala_wcs,                         
+                   bad_fibres_list=bad_fibres_list,      
                    intensity_axis=0,
                    variance_axis=1,
-                   header=koala_header,
+                   header=koala_header,                  
                    fibre_table=koala_fibre_table,
                    info=info
                    )
+    
+    # Plot RSS image if requested  #!!!
+    if vplot(**kwargs): rss_image(rss, **kwargs)
```

Comment:
This should not be here. The output of `rss_image` is never returned, and it will only work in interactive environments (e.g, ipython/spyder). We can create a method within `RSS` that returns this plot and can be called at any time during the reduction/processing. 

I would suggest to remove this lines, and implement the rss_image function as a new method of the `RSS` class.

Reply:
I agree that `rss_image` should be common to all `RSS` objects (i.e. a class method), not just for KOALA.

However, I can imagine, for instance, that a non-interactive pipeline would still want to produce QC plots, and this would be just one of them. The plotting routines should automatically decide whether to show the figure in the screen and/or to save a file. In case of doubt, no screen at all, unless it is specifically requested.

Reply:
`rss_image` can save the plot in a file if requested when argument `save_plot_in_file` is not `None`, although I have to check how this can be done adding figures in the same file if several figures are requested. I found this extremely useful. If you don't want it there, we can delete it, I call it again in my `koalaRSS()`. But this is not only for KOALA, it should work for any kind of instrument.

Reply:
> it should work for any kind of instrument
Therefore, it should be somewhere "general", such as `rss.py`, `data_container.py`, or `plotting/qc_plots`.

In any case, wouldn't a name like "plot_spectra" be more informative than `plot_plot`? Same for `x` and `y`: the could would be more readable if they were called something like "wavelength_list" and "spectra_list".